.gitignore

```diff
@@ -6,6 +6,7 @@ dist
 .gulp
 yarn-error.log
 .serverless
+.env
 
 ### VSCode Projects###
 .vscode
```

Comment:
Will this line get removed after you write the criteria for what should and should not get included with app-specific configurations?

Reply:
I will add an override to it so that the folder is ignored, but a specific file (launch.json) is allowed. The .vscode folder also contains developer-specific customisations to the ide such as colour schemes which should not be committed.

Reply:
```suggestion
.vscode/*
!.vscode/launch.json
```